pkg/test/resourcefixture/testdata/basic/monitoring/v1beta1/monitoringdashboard/monitoringdashboardfull/_generated_export_monitoringdashboardfull.golden

```diff
@@ -85,7 +85,7 @@ spec:
       widget:
         alertChart:
           alertPolicyRef:
-            external: projects/${projectId}/alertPolicies/1719436193634641396
+            external: projects/${projectId}/alertPolicies/1719437683981846099
```

Comment:
Looks like we need a normalize override?

Reply:
Yes, I think this is an earlier PR so should go away in a rebase!